README.md

```diff
@@ -37,6 +37,8 @@ jobs:
       with:
         plugin-file: 'plugin.php'
         upgrade-npm-dependencies: "true"
+        node-version: '20'  # Specify the Node.js version if not lts
+
```

Comment:
extra line

```suggestion
        node-version: '20'  # Specify the Node.js version if not lts
```

Reply:
I wasn't able to commit your suggestion for some reason, but removed the extra line manually